framework/src/main/java/org/checkerframework/framework/util/ContractsUtils.java

```diff
@@ -123,10 +124,12 @@ public static ContractsUtils getInstance(GenericAnnotatedTypeFactory<?, ?, ?, ?>
      * @param contractAnnotation a {@link RequiresQualifier}, {@link EnsuresQualifier}, {@link
      *     EnsuresQualifierIf}, or null
      * @param kind the kind of {@code contractAnnotation}
+     * @param clazz the class to determine the return type
      * @return the contracts expressed by the given annotation, or the empty set if the argument is
      *     null
      */
-    private Set<Contract> getContract(Contract.Kind kind, AnnotationMirror contractAnnotation) {
+    private <T extends Contract> Set<T> getContract(
```

Comment:
/__w/1/s/framework/src/main/java/org/checkerframework/framework/util/ContractsUtils.java:131: warning: no @param for <T>
/__w/1/s/framework/src/main/java/org/checkerframework/framework/util/ContractsUtils.java:166: warning: no @param for <T>

Reply:
Could you please add those? I'm traveling the rest of the day and it would be great to have these changes in the release.

Reply:
I've added the javadoc.